index.js

```diff
@@ -25,15 +25,21 @@
 	var index = 0;
 	var taskStart = 0;
 	var tasklength = 0;
+
+	const getCurrentTime = typeof performance === 'object' && typeof performance.now === 'function'
+	? () => performance.now()
+	: () => Date.now();
```

Comment:
Won't this work without the arrow functions?
```
  ? performance.now
  : Date.now;
```

Reply:
Yes it works, but I added the arrow function to prevent redundant calling